sdk-wasm-js/src/hash.rs

```diff
@@ -7,7 +7,19 @@ use {
 
 #[wasm_bindgen]
 #[derive(Debug, PartialEq, Eq, Clone)]
-pub struct Hash(pub(crate) solana_hash::Hash);
+pub struct Hash {
+    pub(crate) inner: solana_hash::Hash,
+}
+
+impl Hash {
+    pub fn new(inner: solana_hash::Hash) -> Self {
+        Self { inner }
+    }
```

Comment:
I wonder if a

```rust
impl From<solana_hash::Hash> for Hash {}
```

would be more idiomatic here if the behavior is more akin to conversions?